### PR TITLE
docs(generators): Update docs for generators usage

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
## Motivations

Updating the docs where it was mentioning the generators. I added the new select choice the user will have when they use `npm run generate`.

### Added

- 

### Changed

- Updating the docs where it was mentioning the generators. I added the new select choice the user will have when they use `npm run generate`.

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/guides/create-basic-component.stories.mdx [hunk 1]
```diff
@@ -85,6 +93,8 @@ Right, I promised to explain them.
 | `AmountCounter.test.tsx`    | Self-explanatory.                                                                                                                                                                                           |
 | `AmountCounter.css`         | Self-explanatory.                                                                                                                                                                                           |
 | `AmountCounter.css.d.ts`    | This wasn't on the original list but it's very important. This is how react knows what class names you added and automagically serializes them so it doesn't clash with other components. More on it later. |
+| `AmountCounter.stories.mdx` | This file is the front page of the component documentation where you will describe the main features of it.                                                                                                 |
```
Comment (MichaelParadis) [inline docs/guides/create-basic-component.stories.mdx:96]: image
This was already present on line 84

**Review by MichaelParadis — CHANGES_REQUESTED** [review]
Minor change. Looks good besides that

File: docs/guides/create-basic-component.stories.mdx (hunk 1 above)
Comment (rodrigoeidelvein) [reply docs/guides/create-basic-component.stories.mdx]: Oohhh, I didn't even notice. Thanks for catching it. I moved the `Web.stories.tsx` to the line below the `AmountCounter.stories.mdx`

**Review by MichaelParadis — APPROVED** [review]
LGTM!

**Pull request merged**